### ISSUE TITLE
Confidence threshold: Input logits into softmax function to check probability distribution

### ISSUE THREAD

**Issue by sriyavasudevan** [sriyavasudevan/Question-Answering-System#10]
feed logits into the softmax function - check the distribution and see if the highest correspond with softmax output and argmax output

**Comment by sriyavasudevan** [comment]
used softmax function to get the probability distribution and checked that the argmax of the output matches that of the logits. However, the threshold score is still being tested. Thinking about 0.75 but questions based on history has a lower score, so might have to lower it.

**Comment by sriyavasudevan** [comment]
Plot the probability distribution to be able to see the benchmark for the answers. If questions based on history have a lower confidence score, can lower the benchmark from question 2 onwards. @sriyavasudevan

**Comment by sriyavasudevan** [comment]
weigh the start with 0.7 and end score with 0.3 probabilities and sum them up = avg confidence of answer

**Comment by sriyavasudevan** [comment]
completed
